kiwi/bootloader/config/grub2.py

```diff
@@ -610,6 +632,23 @@ def _create_efi_image(self, uuid=None, mbrid=None, lookup_path=None):
             ] + Defaults.get_grub_efi_modules(multiboot=self.xen_guest)
         )
 
+    def _create_efi_config_search(self, uuid=None, mbrid=None):
+        efi_boot_config = self.efi_boot_path + '/grub.cfg'
+        if uuid:
+            self._create_early_boot_script_for_uuid_search(
+                efi_boot_config, uuid
+            )
+        else:
+            self._create_early_boot_script_for_mbrid_search(
+                efi_boot_config, mbrid
+            )
+        with open(efi_boot_config, 'a') as config:
+            config.write(
+                'configfile ($root)/boot/{0}/grub.cfg{1}'.format(
```

Comment:
If $prefix is set correctly, it should be enough to say "configfile grub.cfg". I can't shake off the feeling that calling "normal" is the better option though, but I'll have to double check with Raymund.

Reply:
ok, please clarify with Raymond and I'll give the shortest form a test :) Thanks

Reply:
just calling 'normal' worked for me, I'll update the pull request